.github/workflows/gen_github_actions.py

```diff
@@ -40,7 +40,18 @@
 STRATEGY = od([('fail-fast', 'false')])
 
 EXCLUDE_LIST = []
-
+JIT_EXCLUDE_LIST = [
```

Comment:
this can be a function generating the list

Reply:
Creating the function that generalizes to the different configuration of Exclude List is possible but not necessarily quick, so I would post-pone this